README.md

```diff
@@ -180,6 +180,29 @@ We can always use layer to search for data availability:
   const datesS1 = await layerS1.findDatesUTC(bbox, fromTime, toTime);
 ```
 
+## Getting basic statistics and histogram
+
+Getting basic statistics(mean, min, max, standard deviation) and a histogram for a geometry(Polygon or MultiPolygon).
+The histogram uses the `equalfrequency` binning method and defaults to 5 bins.
+
+```javascript
+  const layerS2L2A = new S2L2ALayer({
+    instanceId: '<my-instance-id>',
+    layerId: 'LAYER_S2L2A',
+    maxCloudCoverPercent: 100,
+  });
+
+const stats = await layerS2L2A.getStatsAndHistogram({
+  geometry: geometry, 
+  crs: CRS_EPSG4326, 
+  fromTime: fromTime, 
+  toTime: toTime, 
+  resolution: resolution,
+  binsAmount:10,
```

Comment:
Nitpick, fftr: `binsAmount` sounds weird to me, maybe `nBins` or just `bins`?

Reply:
changed to `bins`